lighthouse-core/lib/i18n/README.md

```diff
@@ -0,0 +1,305 @@
+they are used so that developers can easily understand their context. It also
+comes with its own syntax.
+
+UIStrings are defined in a simple Object with properties defining the strings.
+
+*   simple string named "title", with no description:
+
+    ```javascript
+    const UIStrings = {
+      title: 'Minify CSS',
+    };
+    ```
+
+*   simple string, with a description above property as a comment:
+
+    ```javascript
+    const UIStrings = {
+      /** Imperative title of a Lighthouse audit that ... */
+      title: 'Minify CSS',
+    };
+    ```
+
+### Markdown
+
+Strings can also contain some limited markdown, that will be rendered in the
+report.
+
+*   Code block.
+
+    To format some text as code it should be contained in `backticks`. These
+    should be used whenever code is non-translatable. Such as HTML tags or
+    snippets of code. Also note that there is no escape character for using
+    backticks as part of the string, so ONLY use backticks to define code
+    blocks.
+
+    Example usage:
+
+    ```javascript
+    const UIStrings = {
+      title: 'Document has a `<title>` element',
+    };
+    ```
+
+*   Link.
+
+    To convert a section of text into a link to another URL, enclose the text
+    itself in [brackets] and then immediately include a link after it in
+    (parenthesis). Note that `[link text] (https://...)` is NOT VALID because of
+    the space and will not be converted to a link.
+
+    Example usage:
+
+    ```javascript
+    const UIStrings = {
+      description: 'The value of ... [Learn More](https://google.com/)',
+    };
+    ```
+
+### ICU Replacement Syntax
+
+ICU syntax is used throughout Lighthouse strings, and they are specified
+directly in UIStrings. They follow 1 of 3 flavors.
+
+*   Direct replacement
+
+    This is simply a direct replacement of text into a string. Often used for
+    Proper Nouns, code, or other text that is dynamic and added at runtime, but
+    requires no additional formatting.
+
+    Example usage:
+
+    ```javascript
+    const UIStrings = {
+        didntCollectScreenshots: `Chrome didn't .... ({errorCode})`,
+    };
+    ```
+
+    ICU replacements must use a JSDoc type syntax to specify an example for
+    direct ICU replacements.
+
+    *   To specify the description, use
+
+        `@description <description>`
+
+    *   To specify an example for an ICU replacement, use
+
+        `@example {<ICU variable name>} <example for ICU replacement>`
+
+    ```javascript
+    const UIStrings = {
+        /**
+         * @description Error message explaining ...
+         * @example {errorCode} NO_SPEEDLINE_FRAMES
+         */
+        didntCollectScreenshots: `Chrome didn't .... ({errorCode})`,
+    };
+    ```
+
+*   Complex replacement
+
+    When more complex numerical ICU replacement is needed the syntax is mostly
+    the same as direct replacement. This is often used when replcaing times, or
+    percentages.
```

Comment:
maybe explain that the numbers are formatted, that's what makes it complex? Or could we even call it "formatted replacement" instead of complex?

Reply:
Done in the ICU section above.